src/makeRobot.js

```diff
@@ -38,7 +38,54 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  return {
+    name: name,
+    wheels: wheels,
+    version: version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    get info() {
+      return `name: ${this.name}, chip version: ${version}, wheels: ${wheels}`;
+    },
+    goForward(move = 1) {
```

Comment:
```suggestion
    },
    
    get info() {
      return `name: ${this.name}, chip version: ${version}, wheels: ${wheels}`;
    },
    
    goForward(move = 1) {
```
Add empty lines between methods to make your code more readdable